airflow/www/extensions/init_views.py

```diff
@@ -304,8 +304,4 @@ def init_api_experimental(app):
 def init_api_auth_provider(connexion_app: connexion.FlaskApp):
     """Initialize the API offered by the auth manager."""
     auth_mgr = get_auth_manager()
-    blueprint = auth_mgr.get_api_endpoints(connexion_app)
-    if blueprint:
-        base_paths.append(blueprint.url_prefix if blueprint.url_prefix else "")
```

Comment:
We do not have any `blueprint` here, so we can not update `base_path` But we need to handle base path update logic somewhere. Whats the impact of removing this line.